packages/vitest/src/integrations/css/css-modules.ts

```diff
@@ -0,0 +1,19 @@
+import type { CSSModuleScopeStrategy } from '../../types'
+
+export function generateCssFilenameHash(filename: string) {
+  return Buffer.from(filename).toString('base64').substring(0, 6)
```

Comment:
This is not a good hash function as different files in the same folder will get the same hash

Reply:
What do you propose instead?

Reply:
Something like md5 that uses entire length of the string

Reply:
We still need to truncate it though to 5-6 chars, otherwise the length might be too big

Reply:
Problem is that base64 is not a hash:

`Buffer.from(filename).toString('base64').substring(0, 6)`:

`folder/file1.module.css` -> `Zm9sZG`  
`folder/file2.module.css` -> `Zm9sZG`

`md5(filename).substring(0, 6)`

`folder/file1.module.css` -> `9d8841`   
`folder/file2.module.css` -> `34c715`

postcss-modules uses [string-hash](https://github.com/darkskyapp/string-hash/blob/master/index.js)

Reply:
updated to use md5